webclient/templates/base_logged_in.html

```diff
@@ -215,10 +215,11 @@ <h2 class="mb-0">
     <div class="collapse navbar-collapse" id="navbarMobile">
       <ul class="navbar-nav ml-auto mt-2 mt-lg-0">
         <li class="nav-item d-lg-none d-xl-block">
-          <span class="nav-link">
+          <span class="nav-link" rel="tooltip" data-placement="top"
+                title="{{ user.email }} ({{ user.hlavni_role }} - {{ user.organizace }})">
```

Comment:
posunes prosim ten radek s title=... o jedno vis at to je spolu s 218 na jednom? Tak to nerozbije ty preklady co jsem delal.